cli/src/printer.rs

```diff
@@ -182,7 +185,12 @@ impl DisplayTable for PrintableSource {
                     None => "none".dimmed(),
                 },
             },
-            self.source.title
+            self.source.title,
+            if let Some(stats) = &self.stats {
+                stats.num_comments.to_string()
+            } else {
+                "N/A".to_string()
```

Comment:
For consistency with above?
```suggestion
                "none".dimmed()
```